diff -Naur a/frameworks/base/data/keyboards/Generic.kl b/frameworks/base/data/keyboards/Generic.kl
--- a/frameworks/base/data/keyboards/Generic.kl	2013-12-10 00:17:08.342290000 +0800
+++ b/frameworks/base/data/keyboards/Generic.kl	2013-12-10 05:45:36.383022639 +0800
@@ -121,7 +121,7 @@
 key 99    SYSRQ
 key 100   ALT_RIGHT
 # key 101 "KEY_LINEFEED"
-key 102   MOVE_HOME
+key 102   HOME
 key 103   DPAD_UP
 key 104   PAGE_UP
 key 105   DPAD_LEFT
@@ -191,7 +191,7 @@
 key 169   CALL
 # key 170 "KEY_ISO"
 key 171   MUSIC
-key 172   HOME
+key 172   MOVE_HOME
 # key 173 "KEY_REFRESH"
 # key 174 "KEY_EXIT"
 # key 175 "KEY_MOVE"
